simulation/deploy_contracts.py

```diff
@@ -13,62 +15,107 @@
 
 # web3.py instance
 w3 = Web3(HTTPProvider('http://127.0.0.1:8545'))
-w3.personal.unlockAccount(w3.eth.accounts[0], '1')
 
-ESCROW_ADDRESS = w3.eth.accounts[1]
-HOLDERS_ACCOUNTS = w3.eth.accounts[1:4]
-
-compiled_source = compile_files([
-    "../contracts/Holder.sol",
-    "../contracts/ICO_controller.sol"], optimize=True)
-
-holder_interface = compiled_source['../contracts/Holder.sol:Holder']
-ico_controller_interface = compiled_source['../contracts/ICO_controller.sol:ICO_controller']
-
-holder_contract = w3.eth.contract(
-    abi=holder_interface['abi'],
-    bytecode=holder_interface['bin'])
-ico_controller_contract = w3.eth.contract(
-    abi=ico_controller_interface['abi'],
-    bytecode=ico_controller_interface['bin'])
-
-holder_tx_hash = holder_contract.deploy(
-    transaction={'from': w3.eth.accounts[0]},
-    args=(HOLDERS_ACCOUNTS,  # List of accounts that control holder account
-          1,  # Number of accounts needed to confirm changes
-          ESCROW_ADDRESS)  # Escrow account
-)
-
-holder_receipt = wait_for_tx(holder_tx_hash,
-                             w3,
-                             wait_message="Wait for Holder contract to be deployed")
-
-
-holder_contract_address = holder_receipt['contractAddress']
+try:
+    w3.personal.unlockAccount(w3.eth.accounts[0], '1')
+except:
+    pass
 
-controller_tx_hash = ico_controller_contract.deploy(
-    transaction={'from': w3.eth.accounts[0]},
-    args=(holder_contract_address,
-          ESCROW_ADDRESS)  # Escrow account
-)
 
-controller_receipt = wait_for_tx(controller_tx_hash,
+def ethereum_accounts(accounts):
```

Comment:
For this check it is better to use the functions http://web3py.readthedocs.io/en/latest/overview.html#overview-addresses
including that you can correct its, if the addresses is not "Checksum"